ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/libvirt.py

```diff
@@ -61,7 +61,7 @@ def collect(self, device, log):
             'port': device.zCommandPort,
             'user': device.zCommandUsername,
             'password': device.zCommandPassword,
-            'identities': ['~/.ssh/id_rsa', '~/.ssh/id_dsa'],
+            'identities': [device.zKeyPath],
```

Comment:
I don't think this will work unless you also add zKeyPath to libvirt.deviceProperties first.